now.json

```diff
@@ -4,7 +4,8 @@
     },
     "build": {
         "env": {
-            "AIRTABLE_TOKEN": "@airtable_token"
```

Comment:
Why delete that?

Reply:
No need to put the token on the front side